packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -101,6 +102,11 @@ const DATA_MAP: AuthDataMap = {
         callback: () => {
             cookieStorage.delete('cc-nx-g')
         }
+    },
+    site_id: {
+        // do we need this be a cookie to support plugin_slas?
```

Comment:
Do we expect there will be hybrid launches using this package in the future? If yes, we probably want to support plugin_slas?